textproc/cfonts/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo  1.0
+
+github.setup        DominikWilkowski cfonts 1.0.2rust v
+revision            0
+
+categories          textproc
+license             GPL-3+
+maintainers         {DominikWilkowski @DominikWilkowski}
```

Comment:
Please also add `openmaintainer`, unless there is a specific reason why you and only you should make changes to this port:
```
maintainers         {DominikWilkowski @DominikWilkowski} \
                    openmaintainer
```

Reply:
added